test/net_http_test.rb

```diff
@@ -396,6 +432,12 @@ def test_persistent_state_after_server_restart
 
   private
 
+  def half_open_cicuit!(backwards_time_travel = 10)
```

Comment:
@sirupsen I wanted to use the CircuitBreakerHelper but the `half_open_circuit!` method uses a `open_circuit!` method with an argument and when I call `half_open_circuit!` from the `net_http_test` file, it calls the `open_circuit!` method inside `net_http_test` instead of the `open_circuit!` method inside the helper which raises. To make it work, I would have to change the `open_circuit!` method name everywhere. What do you think of simply keeping the `half_open_cicuit!` method in the `net_http_test` file?

Reply:
That's fine with me.